examples/chart/teleport-kube-agent/templates/NOTES.txt

```diff
@@ -20,3 +20,21 @@ You will face compatibility issues trying to run a different Teleport version wi
 If you want to run Teleport version {{.Values.teleportVersionOverride}},
 you should use `helm --version {{.Values.teleportVersionOverride}}` instead.
 {{- end }}
+{{- if contains "-gke." .Capabilities.KubeVersion.Version -}}
+{{- $groupName := (coalesce .Values.adminClusterRoleBinding.name "cluster-admin") }}
+
+WARNING: GKE Autopilot clusters forbid users from impersonating system-wide identities.
+This means you won't be able to use the `system:masters` Kubernetes Group in
+the Teleport Roles for GKE Autopilot clusters.
+
+Given that you installed Teleport on a GKE cluster, we recommend you use the
+Kubernetes Group `{{ $groupName }}` instead of `system:masters` in the Teleport Roles
+for GKE Autopilot clusters.
```

Comment:
Can we also include a role snippet example below showing appropriate `kubernetes_groups: ["{{ $groupName }}"]` rule in a role for clarity?